src/main/java/org/broadinstitute/hellbender/tools/spark/sv/GATKSVVCFHeaderLines.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineCount;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GATKSVVCFHeaderLines {
+
+    public static Map<String, VCFHeaderLine> vcfHeaderLines = new HashMap<>();
+
+    // VCF standard SV header lines
+    // todo: add these and the other standard SV info fields from the VCF spec to htsjdk VCFStandardHeaderLines
+    public static final String SVTYPE = "SVTYPE";
+    public static final String SVLEN = "SVLEN";
+
+    // GATK-SV specific header lines
+    public static final String TOTAL_MAPPINGS = "TOTAL_MAPPINGS";
+    public static final String HQ_MAPPINGS = "HQ_MAPPINGS";
+    public static final String MAPPING_QUALITIES = "MAPPING_QUALITIES";
+    public static final String ALIGN_LENGTHS = "ALIGN_LENGTHS";
+    public static final String MAX_ALIGN_LENGTH = "MAX_ALIGN_LENGTH";
+    public static final String BREAKPOINT_IDS = "BREAKPOINT_IDS";
```

Comment:
Sorry to find this late, but this is probably better to be renamed to "ASSEMBLY_IDS" as well?